src/Kafka/Consumer/ConsumerProperties.hs

```diff
@@ -54,12 +55,12 @@ data ConsumerProperties = ConsumerProperties
   { cpProps            :: Map Text Text
   , cpLogLevel         :: Maybe KafkaLogLevel
   , cpCallbacks        :: [Callback]
-  , cpCallbackPollMode :: CallbackPollMode
+  , cpCallbackPollMode :: Maybe CallbackPollMode
   }
 
 instance Sem.Semigroup ConsumerProperties where
-  (ConsumerProperties m1 ll1 cb1 _) <> (ConsumerProperties m2 ll2 cb2 cup2) =
-    ConsumerProperties (M.union m2 m1) (ll2 `mplus` ll1) (cb1 `mplus` cb2) cup2
+  (ConsumerProperties m1 ll1 cb1 cup1) <> (ConsumerProperties m2 ll2 cb2 cup2) =
+    ConsumerProperties (M.union m2 m1) (ll2 `mplus` ll1) (cb1 `mplus` cb2) (cup1 `mplus` cup2)
```

Comment:
Shouldn't it be  ```(cup2 `mplus` cup1)``` instead? Otherwise it'll be impossible to override once set...